index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+
+<body>
+
+    <nav>
+        <h1>Dr. Borlaug
+        </h1>
+        <h1>The man who saved a billion lives</h1>
+    </nav>
+
+    <div class="background-image">
+        <img src="Images/norman_borlaug_and_wheat_breeding_trainees_1972_1.jpg" alt="">
+        <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+    </div>
+
+    <div class="biography">
+        <h3>Here's a time line of Dr. Borlaug's life:</h3>
+        <ul>
+            <li> <b>&#8226 1914</b> - Born in Cresco, Iowa</li><br>
+            <li>&#8226 <b>1933</b> - Leaves his family's farm to attend the University of Minnesota, thanks<br> to a Depression era program known as the "National Youth Administration"<br></li>
+            <br>
+            <li>&#8226 <b>1935</b> - Has to stop school and save up more money. Works in the Civilian<br> Conservation Corps, helping starving Americans. "I saw how food changed<br> them", he said. "All of this left scars on me."<br></li>
+            <br>
```

Comment:
it is better to use width than
tag

Reply:
- [OPTIONAL] Nice one @AbdifatahYasin1, but try as much as you can to include `WHY` the student is asked to make use of a suggestion as this would make it clearer for the student to know why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where more information could be got for  more clarification.